insights/parsers/tests/test_fstab.py

```diff
@@ -99,3 +99,30 @@ def test_fstab():
     assert sitedata_mount.fs_mntops['defaults'] is True
     assert sitedata_mount.fs_vfstype == "nfs"
     assert sitedata_mount.fs_spec == "192.168.48.65:/cellSiteData"
+
+
+FSTAB_DEVICE_PATH_TEST_INFO = """
+/dev/sda2                    /                          ext4    defaults        1 1
+/dev/sdb2                    /var                       ext4    defaults        1 1
+/dev/sdb3                    /var/crash                 ext4    defaults        1 1
+/dev/mapper/VolGroup-lv_usr  /usr                       ext4    defaults        1 1
+UUID=qX0bSg-p8CN-cWER-i8qY-cETN-jiZL-LDt93V /kdump                   ext4    defaults        1 2
+/dev/mapper/VolGroup-lv_swap swap                    swap    defaults        0 0
+proc                    /proc                   proc    defaults        0 0
+/dev/mapper/vgext-lv--test      /lv_test        ext3    defaults        0       0
+""".strip()
+
+
+def test_match_device_of_path():
+    fstab_info = fstab.FSTab(context_wrap(FSTAB_DEVICE_PATH_TEST_INFO))
+    path_device_map = {'/var/crash': '/dev/sdb3',
+                       '/var/some/path': '/dev/sdb2',
+                       '/kdump/crash': 'UUID=qX0bSg-p8CN-cWER-i8qY-cETN-jiZL-LDt93V',
+                       '/some/path': '/dev/sda2',
+                       '/lv_test': '/dev/mapper/vgext-lv--test',
+                       '/lv': None,
```

Comment:
Is the `/lv` different with `/some/path`? The result of `/lv` should be `/dev/sda2`, right?

Reply:
Yes, you are right. I will change the function's logic.